lib/windows.js

```diff
@@ -119,9 +124,58 @@ function windows() {
 	// Remove null characters from buffer
 	const processFileNameBufferClean = ref.reinterpretUntilZeros(processFileNameBuffer, wchar.size);
 	// Get process file path as a string
-	const processPath = wchar.toString(processFileNameBufferClean);
+	let processPath = wchar.toString(processFileNameBufferClean);
 	// Get process file name from path
-	const processName = path.basename(processPath);
+	let processName = path.basename(processPath);
+	let newProcessName = null;
+	let newProcessPath = null;
```

Comment:
```suggestion
	let newProcessPath;
```

Better to just use `undefined` instead of `null`.